src/StreamJsonRpc.Desktop/Resources.resx

```diff
@@ -121,6 +121,14 @@
     <value>Both readable and writable are null.</value>
     <comment>'readable' and 'writable' are parameter names and should not be localized.</comment>
   </data>
+  <data name="ConflictingMethodAttributeValue" xml:space="preserve">
+    <value>The {0} Name value of {1} maps to multiple methods.</value>
+    <comment>{0} is the attribute name {1} is the attribute value.</comment>
+  </data>
+  <data name="ConflictingMethodNameAttribute" xml:space="preserve">
+    <value>The '{0}' method has multiple {1} defined with different values.</value>
```

Comment:
I think this could be more clearly actionable to the user if it were reworded to:

> All overloads of the "{type}.{method}" method must share a common value for JsonRpcMethodAttribute.Name.